lib/spack/spack/relocate.py

```diff
@@ -240,44 +230,63 @@ def needs_text_relocation(filetype):
     """
     Check whether the given filetype is text that may need relocation.
     """
-    return ("text" in filetype)
+    return (" text" in filetype)
```

Comment:
In the ROOT package there is a library called libmathtext.a. File type always has a leading space on text.

Reply:
> In the ROOT package there is a library called libmathtext.a

Wouldn't we be talking about the type of libmathtext.a vs. its name?

> File type always has a leading space on text

OK in that case this is fine as is (the logic that is - it would still be good to see an example in a comment)